### PR TITLE
Update index.md repl to csb

### PR DESCRIPTION
**Pull request by ptrtorain**
//TODO

fork this repository https://codesandbox.io/s/github/rainyEra/reatom-react-ts-search-example and change replit link to csb. This csb is copy of your reatom-react-ts-search-example replit

### CODE REVIEW

**Comment by artalar** [comment]
WIP: https://codesandbox.io/s/reatom-react-hello-world-fgt6jn

**Comment by ptrtorain** [comment]
thanks!

File: docs/src/pages/index.md [hunk 1]
```diff
@@ -29,9 +29,9 @@ Let's define input state and compute a greeting from it.
 npm i @reatom/core
 ```
 
-[vanilla repl](https://codesandbox.io/s/reatom-first-example-6oo36v?file=/src/index.ts)
+[vanilla codesandbox](https://codesandbox.io/s/reatom-first-example-6oo36v?file=/src/index.ts)
 
-[react repl](https://replit.com/@artalar/reatom-react-ts)
+[react codesandbox](https://codesandbox.io/s/github/artalar/reatom-react-ts?file=/src/App.tsx)
```
Comment (artalar) [inline docs/src/pages/index.md]:
```suggestion
[react codesandbox](https://codesandbox.io/s/reatom-react-hello-world-fgt6jn?file=/src/model.ts)
```

File: docs/src/pages/index.md [hunk 2]
```diff
@@ -29,9 +29,9 @@ Let's define input state and compute a greeting from it.
 npm i @reatom/core
 ```
 
-[vanilla repl](https://codesandbox.io/s/reatom-first-example-6oo36v?file=/src/index.ts)
+[vanilla codesandbox](https://codesandbox.io/s/reatom-first-example-6oo36v?file=/src/index.ts)
```
Comment (artalar) [inline docs/src/pages/index.md]:
```suggestion
[vanilla codesandbox](https://codesandbox.io/s/reatom-vanila-hello-world-6oo36v?file=/src/index.ts)
```

File: docs/src/pages/index.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ This example is close to real life and shows the complexity of interactive UI. I
 npm i @reatom/framework @reatom/npm-react
 ```
 
-[repl](https://replit.com/@artalar/reatom-react-ts-search-example#src/App.tsx)
+[codesandbox](https://codesandbox.io/s/reatom-react-hello-world-fgt6jn)
```
Comment (artalar) [inline docs/src/pages/index.md]:
```suggestion
[codesandbox](https://codesandbox.io/s/reatom-react-search-component-l4pe8q?file=/src/App.tsx)
```

**Pull request merged**